src/main/java/org/reaktivity/nukleus/socks/internal/SocksController.java

```diff
@@ -1,17 +1,16 @@
 /**
  * Copyright 2016-2017 The Reaktivity Project
  *
```

Comment:
Why modify the license header section?  Please keep this the same for all reaktivity